src/ape/cli/options.py

```diff
@@ -145,6 +145,24 @@ def skip_confirmation_option(help=""):
     )
 
 
+def upgrade_confirmation_option(help=""):
```

Comment:
I am not sure we should put this here. This option seems only really specific to the `ape_plugins` plugin, so maybe we just define in the normal `_cli.py` there.

Reply:
so i should only define this method in the _cli?